documentation/modules/proc_storage-copy-offload.adoc

```diff
@@ -0,0 +1,215 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: PROCEDURE
+[id="proc_storage-copy-offload_{context}"]
+= Migrating {vmw} vSphere VMs by using storage copy offload
+
+[role="_abstract"]
+You can use the storage copy offload feature of {project-first} to migrate {vmw} vSphere virtual machines (VMs) faster than by other methods. 
+
+.Prerequisites
+
+In addition to the regular xref:vmware-prerequisites_mtv[{vmw} prerequisites], storage copy offload has the following additional prerequisites: 
+
+* A configured Dell PowerFlex Storage Data Client (SDC) or one of the following:
+** A configured Internet Small Computer System Interface (iSCSI) storage systdem
+** A configured Fibre Channel storage system
+* A working Container Storage Interface (CSI) driver connected to the above and to {virt}
+* A configured {vmw} vSphere provider
+* vSphere users must have a role that includes thw following privileges (suggested name: `StorgeOffloader`):
+
+** Global
+*** Settings
+** Datastore
+*** Browse datastore
+*** Low level file operations
+** Host Configuration
+*** Advanced settings
+*** Query patch
+*** Storage partition configuration
+
+.Procedure
+
+. In the {project-short} Operator, set the value of `feature_copy_offload` to `true` in `forklift-controller` by running the following command:
++
+[source, terminal]
+----
+oc patch forkliftcontrollers.forklift.konveyor.io forklift-controller --type merge -p '{"spec": {"feature_copy_offload": "true"}}' -n openshift-mtv
+----
+
+. Create a `Secret` in the namespace in which the migration provider is set up, usually `openshift-mtv`. Include the following credentials in the `Secret`. Note that Hitachi has a different set of credentials:
++
+[cols="4", options="header"]
+.Credentials for a non-Hitachi storage copy offload Secret
+|===
+|Key| Description| Mandatory?| Default
+
+| `STORAGE_HOSTNAME`
+| IP or URL of the host (string)
+| Yes
+| NA
+
+| `STORAGE_USERNAME`
+| The user's name (string)
+| Yes
+| NA
+
+| `STORAGE_PASSWORD`
+| The user's password (string)
+| Yes
+| NA
+
+| `STORAGE_SKIP_SSL_VERIFICATION`
+| If set to `true`, SSL verification is not performed (`true`, `false`).
+| No
+|`false`
+|===
++
+[cols="4", options="header"]
+.Credentials for a Hitachi storage copy offload Secret
+|===
+|Key| Description| Mandatory?| Default
+
+| `GOVMOMI_HOSTNAME`
```

Comment:
line 75 - 104 are not needed as they are populated by the system automatically (I'll fix the upstream doc as well)